### PR TITLE
cleanup: Move all license headers in C++ source files to SPDX; run clang-format.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/TokTok/qTox/55)

### CODE REVIEW

**Review by robinlinden — DISMISSED** [review]

File: src/appmanager.cpp [hunk 1]
```diff
@@ -418,19 +398,19 @@ void AppManager::cleanup()
...
-            fclose(f);
-    #if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
-            logFileFile.storeRelaxed(nullptr); // atomically disable logging to file
-    #else
-            logFileFile.store(nullptr); // atomically disable logging to file
-    #endif
-        }
-    #endif
+#ifdef LOG_TO_FILE
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```
Comment (Green-Sky) [inline src/appmanager.cpp:402]: I think we can drop these.

File: src/widget/chatformheader.cpp [hunk 2]
```diff
@@ -44,44 +30,43 @@ const short BUTTONS_LAYOUT_HOR_SPACING = 4;
...
-    QString{},
-    QStringLiteral("green"),
-    QStringLiteral("red"),
-    QStringLiteral("yellow"),
-    QStringLiteral("yellow"),
+    QString{},                //
+    QStringLiteral("green"),  //
+    QStringLiteral("red"),    //
+    QStringLiteral("yellow"), //
+    QStringLiteral("yellow"), //
```
Comment (Green-Sky) [inline src/widget/chatformheader.cpp:37]: why?

Comment (iphydf) [reply src/widget/chatformheader.cpp:37]: So clang-format doesn't put some of them on the same line.

Comment (Green-Sky) [reply src/widget/chatformheader.cpp:37]: meh, fine

**Review by Green-Sky — APPROVED** [review]

**Pull request merged**